src/mmw/js/src/modeling/gwlfe/entry/views.js

```diff
@@ -76,6 +112,110 @@ var TabContentsView = Marionette.CollectionView.extend({
     },
 });
 
+var FieldView = Marionette.ItemView.extend({
+    className: 'row',
+    template: fieldTmpl,
+
+    ui: {
+        input: 'input',
+    },
+
+    events: {
+        'change @ui.input': 'updateUserValue',
+        'click button': 'resetUserValue',
+    },
+
+    modelEvents: {
+        'change:userValue': 'render',
+    },
+
+    onRender: function() {
+        this.$('[data-toggle="popover"]').popover();
+    },
+
+    updateUserValue: function() {
+        var value = this.ui.input.val();
+
+        if (value) {
+            this.model.set('userValue', value);
+        } else {
+            this.model.set('userValue', null);
+        }
+    },
+
+    resetUserValue: function() {
+        this.ui.input.val('');
+        this.$('[data-toggle="popover"]').popover('destroy');
+        this.model.set('userValue', null);
+    }
+});
+
+var FieldsView = Marionette.CollectionView.extend({
+    childView: FieldView,
+});
+
+var SectionView = Marionette.LayoutView.extend({
+    template: sectionTmpl,
+
+    regions: {
+        fieldsRegion: '.rows',
+    },
+
+    onShow: function() {
+        this.fieldsRegion.show(new FieldsView({
+            collection: this.model.get('fields'),
+        }));
+    }
+});
+
+var SectionsView = Marionette.CollectionView.extend({
+    className: 'section',
+
+    childView: SectionView,
+});
+
+function showSettingsModal(title, dataModel, modifications, addModification) {
+    var tabs = new models.EntryTabCollection([
+            // { name: 'efficiencies', displayName: 'Efficiencies' },
```

Comment:
Commenting here to make sure this was intentionally left in albeit commented-out.

Reply:
(Same q about the "animals" and "other" sections below -- I assume the answer's the same too).

Reply:
That is correct. These tabs will have their sections and fields filled in #2960, #2961, and #2962.